install/kubernetes/helm/Istio/values.yaml

```diff
@@ -0,0 +1,13 @@
+Istio:
+   auth_enabled: false 
+   initializer_enabled: false
+   mixer_enabled: true
+   pilot_enabled: true
+   proxy_enabled: true
+   namespace: istio-system
+   ca_hub: default
+   ca_tag: default
+   ca_demo: false 
```

Comment:
What does this do?

Reply:
CA can work in two modes according to info I dug up. demo mode, so CA works within a single namespace and production mode when CA covers all namespaces. Please correct me if I am wrong here. ca_demo just turns on/off demo mode as templates used for these two modes are different